devops/scripts/benchmarks/options.py

```diff
@@ -40,7 +40,7 @@ class Options:
     build_compute_runtime: bool = False
     extra_ld_libraries: list[str] = field(default_factory=list)
     extra_env_vars: dict = field(default_factory=dict)
-    compute_runtime_tag: str = "25.09.32961.8"
+    compute_runtime_tag: str = "25.13.33276.10"
```

Comment:
did you check if this version is working on our runner...? 😉  (as I understand it's not tested per PR)

Reply:
I checked on dnp-02, it works fine.